guides/hack/11-built-in-types/36-enum-class-label.md

```diff
@@ -0,0 +1,96 @@
+## Values v. Bindings
+
+With [enum types](enumerated-types) and [enum classes](enum-class), most of the focus is given to their values.
+Expressions like `E::A` denote the value of `A` in `E`, but the fact that `A` was used to access it is lost.
```

Comment:
I could be wrong, but did you mean to say that the fact that the enum `E` was used to access the value at `A' is lost? Or is it correct to say that `A` is lost because the value at `A` is only cared about.

(Same note for line 24)

Reply:
We can "keep" the fact about `E` is kept a bit: you can have a `function f(E $x)` and pass in `E::A`. However in `f`, anything about `A` is gone